catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityFilterPlugin.java

```diff
@@ -32,42 +45,37 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.MapUtils;
+import org.opengis.filter.Filter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-public class MetacardValidityFilterPlugin implements PolicyPlugin {
+/**
+ * A {@link ddf.catalog.plugin.PreFederatedQueryPlugin} and {@link PolicyPlugin} which performs
```

Comment:
It extends the abstract class that is a PreFederatedQueryPlugin. I put this here since I think it helps clarifies exactly when this plugin will be running, but if it's more confusing I can change it.